.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           command: |
             source /opt/rh/gcc-toolset-9/enable
             cd presto-native-execution
-            cmake -B _build/debug -GNinja -DTREAT_WARNINGS_AS_ERRORS=1 -DENABLE_ALL_WARNINGS=1 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
+            cmake -B _build/debug -GNinja -DTREAT_WARNINGS_AS_ERRORS=1 -DENABLE_ALL_WARNINGS=1 -DCMAKE_BUILD_TYPE=Debug -DPRESTO_ENABLE_PARQUET=ON -DCMAKE_PREFIX_PATH=/usr/local -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
```

Comment:
This change is already submitted in https://github.com/prestodb/presto/commit/46dd008becf6ce84d16b9c18eff2f46a9db7e72c. Please can you rebase. That should help with the merge.

Reply:
Done. Thanks for letting me know.